homeassistant/components/xiaomi/device_tracker.py

```diff
@@ -113,7 +113,7 @@ def _retrieve_list(host, token, **kwargs):
     url = "http://{}/cgi-bin/luci/;stok={}/api/misystem/devicelist"
     url = url.format(host, token)
     try:
-        res = requests.get(url, timeout=5, **kwargs)
+        res = requests.get(url, timeout=30, **kwargs)
```

Comment:
The scan interval for this platform is every 12 seconds.
How does that affect this when it takes e.g., 20 seconds for a response?

Should this be adjusted to 10 seconds for starters?

Reply:
Ok, I have changed it from 30s to 10s.